### PR TITLE
Jurijs branch

### PR DESCRIPTION
**Pull request by jdm-de**
# Conflicts:
#	README.md

### CODE REVIEW

**Comment by Mulchmann** [comment]
Hm

File: README.md [hunk 1]
```diff
@@ -4,3 +4,5 @@ This is a temporary repository
 Meine erste Änderung
 
 Meine zweite Änderung
+
+Jkhadkfgjskldjfkl
```
Comment (Mulchmann) [inline README.md]: Hm hm hm

Comment (Mulchmann) [inline README.md]: isich gut

**Comment by Mulchmann** [comment]
Voll gut ey

File: README.md (hunk 1 above)
Comment (krautzberger) [inline README.md]: Jkhadkfgjskldjfkl

Comment (krautzberger) [inline README.md]: Jurij Diaz M... added some commits an hour ago
		Mal gucken, ob's funktioniert 			801d82d
		Merge branch 'master' into jurijs-branch
			1d471f0
@Mulchmann
Member
Mulchmann commented 29 minutes ago

Hm
@Mulchmann
Mulchmann approved these changes 26 minutes ago
README.md
@@ -4,3 +4,5 @@ This is a temporary repository
 Meine erste Änderung
 
 Meine zweite Änderung
+
+Jkhadkfgjskldjfkl
@Mulchmann
Mulchmann 26 minutes ago Member

Hm hm hm
@Mulchmann
Mulchmann 26 minutes ago Member

isich gut

**Pull request closed without merging**